### PR TITLE
fix: Add note about operators installation

### PR DESCRIPTION
**Pull request by cbuto**
## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cwyl02 — APPROVED** [review]

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/kafka-operator/index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ excerpt: Information about the Kafka Operator
 
 Follow these steps to install the Kafka operator in a workspace. This procedure results in a Kafka operator running in a workspace namespace, ready to manage and create Kafka clusters in any project namespaces. See the [Kafka custom resource documentation](../../../../../projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/) for more information on creating Kafka clusters.
 
+<p class="message--note"><strong>NOTE: </strong>You should only install the Kafka operator once per workspace.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/kafka-operator/index.md:17]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Only install the Kafka operator once per workspace.</p>
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ The Kubernetes Operator for Apache Spark aims to make specifying and running Spa
 
 You can find generic installation instructions for workspace catalog applications on the [Application Deployment topic](../../application-deployment).
 
+<p class="message--note"><strong>NOTE: </strong>You should only install the Spark operator once per workspace.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md:19]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Only install the Spark operator once per workspace.</p>
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/zookeeper-operator/index.md [hunk 3]
```diff
@@ -14,6 +14,8 @@ excerpt: Information about the ZooKeeper Operator
 
 Follow these steps to install the ZooKeeper operator in a workspace. This procedure results in a ZooKeeper operator running in a workspace namespace, ready to manage and create ZooKeeper clusters in any project namespaces. See the [ZooKeeper custom resource documentation](../../../../../projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/) for more information on creating ZooKeeper clusters.
 
+<p class="message--note"><strong>NOTE: </strong>You should only install the ZooKeeper operator once per workspace.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/zookeeper-operator/index.md:17]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Only install the ZooKeeper operator once per workspace.</p>
```

**Review by cmastr — APPROVED** [review]

**Review by gracedo — COMMENTED** [review]
Not related to these changes here, but can we also add a note to the project CR pages that those commands need to be run on *each attached cluster in the project* since iirc it didn't specify that which might be confusing

**Comment by cbuto** [comment]
> Not related to these changes here, but can we also add a note to the project CR pages that those commands need to be run on _each attached cluster in the project_ since iirc it didn't specify that which might be confusing

@gracedo added a note in 94343a5a6b24660cad3dcbd17996028e6e13ed27, let me know what you think!

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md (hunk 2 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md]: fixed in 7195cc0c82201695a22e0befdc2b61ff9f2661c8

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/zookeeper-operator/index.md (hunk 3 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/zookeeper-operator/index.md]: fixed in 7195cc0c82201695a22e0befdc2b61ff9f2661c8

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 4]
```diff
@@ -14,6 +14,8 @@ After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCl
 
 ## Example deployment
 
+<p class="message--note"><strong>NOTE: </strong>The Custom Resources need to be applied to each attached cluster in a project.</p>
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md:17]: https://github.com/mesosphere/dcos-docs-site/pull/4016/files#diff-0dc2b5b15bb1aae5a27cc7f699afcf39280a74ef0cf4dde2b67fec4324ac42f8R13
>After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. Refer to the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.

Maybe in the paragraph above we can also add
```
...by applying a `KafkaCluster` custom resource **on each attached cluster** in the project's namespace.
```

Comment (gracedo) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md:17]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>The Custom Resources must be applied to each attached cluster in a project.</p>
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/index.md [hunk 5]
```diff
@@ -16,6 +16,8 @@ A [Helm chart](https://github.com/pravega/zookeeper-operator/tree/master/charts/
 
 ### Example deployment
 
+<p class="message--note"><strong>NOTE: </strong>The Custom Resources need to be applied to each attached cluster in a project.</p>
```
Comment (gracedo) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/index.md:19]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>The Custom Resources must be applied to each attached cluster in a project.</p>
```

Same comment here re tweaking paragraph above

Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/index.md]: fixed in d58e0411b0e74dbe4797a6be8f8cc146c1c1019e!

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 4 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md]: good idea, fixed in d58e0411b0e74dbe4797a6be8f8cc146c1c1019e!

**Review by cmastr — APPROVED** [review]

**Pull request merged**